docs/source/dev-guide/specs/solver-state.md

```diff
@@ -1,18 +1,16 @@
----
-orphan: true
----
-
-
 ```{admonition} Work in progress
 This page of the documentation is not yet finished and only contains a draft of the content.
 ```
 
-(solver_state_specification)=
-
 # Technical specification: solver state
 
-> Note: This document is a technical specification, which might not be the best way to learn about
-> how the solver works. For that, refer to {ref}`deep_dive_install` and {ref}`deep_dive_solvers`.
+:::{admonition} Note
+:class: warning
+
+This document is a technical specification, which might not be the best way to learn about
+how the solver works. For that, refer to {doc}`deep-dives/install` and
```

Comment:
This link is broken on the preview ☹️ https://continuumio-conda-docs--12041.com.readthedocs.build/projects/conda/en/12041/dev-guide/specs/solver-state.html

Not sure how to fix it. Probably just a MD vs. RST thing.

Reply:
Huh, weird! That works for me locally. Looking into it..